.jazzy.yaml

```diff
@@ -99,6 +100,7 @@ custom_categories:
       - EContextATMVoucherAction
       - EContextStoresVoucherAction
       - OXXOVoucherAction
+      - BACSAction
```

Comment:
Can we give it a bit more generic name?
`ConfirmationAction` or something?

Reply:
I feel this Action is quite specific to BACS to have a generic name.

Reply:
Valid point from Vlad but not sure if this is specific to BACS or not.

Reply:
As far as we know, there isn't any other action with similar properties.